lms/envs/common.py

```diff
@@ -1578,16 +1598,40 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 ################################# WIKI ###################################
 from lms.djangoapps.course_wiki import settings as course_wiki_settings  # pylint: disable=wrong-import-position
 
+# .. toggle_name: WIKI_ACCOUNT_HANDLING
+# .. toggle_implementation: Django setting
+# .. toggle_default: True
+# .. toggle_description: This setting allows the access to Sign up, login and logout views.
+# .. toggle_warnings: Turn to False only if you have your own account handling
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: None
+# .. toggle_tickets: None
 WIKI_ACCOUNT_HANDLING = False
 WIKI_EDITOR = 'lms.djangoapps.course_wiki.editors.CodeMirror'
 WIKI_SHOW_MAX_CHILDREN = 0  # We don't use the little menu that shows children of an article in the breadcrumb
+# .. toggle_name: WIKI_ANONYMOUS
+# .. toggle_implementation: Django setting
+# .. toggle_default: True
+# .. toggle_description: This setting treats none logged in users as the ¨other¨ user group.
+# .. toggle_warnings: In False it doesn´t allow to access anonymous users.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: None
+# .. toggle_tickets: None
 WIKI_ANONYMOUS = False  # Don't allow anonymous access until the styling is figured out
 
 WIKI_CAN_DELETE = course_wiki_settings.CAN_DELETE
 WIKI_CAN_MODERATE = course_wiki_settings.CAN_MODERATE
 WIKI_CAN_CHANGE_PERMISSIONS = course_wiki_settings.CAN_CHANGE_PERMISSIONS
 WIKI_CAN_ASSIGN = course_wiki_settings.CAN_ASSIGN
-
+# .. toggle_name: WIKI_USE_BOOTSTRAP_SELECT_WIDGET
+# .. toggle_implementation: Django setting
+# .. toggle_default: True
+# .. toggle_description: This setting allows you to use bootstrap and you can switch off
```

Comment:
We recommend you leave this off 
```suggestion
# .. toggle_description: This setting allows you to use bootstrap and you can switch off
# .. toggle_description: Enabling this will use the bootstrap select widget. For the original docs, see:
#   https://github.com/edx/django-wiki/blob/edx_release/wiki/conf/settings.py
```